articles/quickstart/native/android/06-calling-apis.md

```diff
@@ -102,16 +104,16 @@ Tell the client to create a new `Call` with the request you created. Call the `e
 
 client.newCall(request).enqueue(new Callback() {
     @Override
-    public void onFailure(Request request, IOException e) {
-        //show error
+    public void onFailure(Request request, final IOException e) {
```

Comment:
Any reason to change this to `final` here?

Reply:
If I remember correctly, that's how Android Studio generates the callback implementation. I can change it back and remove the `final`, what do you think?